control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -1259,8 +1254,8 @@ func TestRun_SecondaryDatacenter_PoliciesAndBindingRulesForACLLogin_NamespacesEn
 				require.NoError(t, err)
 				require.NotNil(t, rb)
 				found = false
-				for x := range rb {
-					if rb[x].BindName == c.Roles[i] {
+				for j := range rb {
+					if rb[j].BindName == c.Roles[i] {
 						found = true
```

Comment:
this test is the same as when namespaces are disabled (OSS). Is there anything ns-specific we should assert on here? should we check here that e.g. these are always created in the default ns because the binding rule creation ignores namespace rules like mirroring and destination ns.

Reply:
That is a good point. I dont mind adding a different assertion here, but maybe the best idea is to not have this test duplicated and only have it run in OSS

Reply:
yeah maybe we don't need the roles checking since they are always set up the same with and w/o namespaces, but I think we can check that we skip namespace flags for the binding rule creation since there's logic there to `skipNamespacing`. I think that also means that we need to add multiple cases here for mirroring and destination ns to check that we always ignore those settings. wdyt?

Reply:
ooh. that is true. let me take a crack at this!